protocol/v2/ssv/validator/committee.go

```diff
@@ -21,6 +22,10 @@ import (
 	"github.com/ssvlabs/ssv/protocol/v2/ssv/runner"
 )
 
+var (
+	maxRunnerAgeSlot = phase0.Slot(32)
```

Comment:
lets try to come up with consistent naming for this new concept

for example `runnerExpirySlots` and `stopExpiredRunners`

Reply:
Agree. Renamed old -> expired